### ISSUE TITLE
New freature: background functionality

### ISSUE THREAD

**Issue by confile** [MHaroonBaig/MotionKit#4]
Are you still working on this feature?

**Comment by MHaroonBaig** [comment]
Yes @confile I'm still working on this feature but iOS restricts MotionKit to run in the background for too long as it keeps the sensor hardware engaged for a bit of time and gets terminated. I'm open to any kind of pull requests regarding this feature including background functionality. 
Or, we can somehow work on it together to have some workaround. I'll leave this issue opened.

**Comment by wittles2324** [comment]
I did add update location in background to but the Accelerometer is not working on background. I tested on iOS9. Do you get success to receive Accelerometer in background on iOS9?

**Comment by shindejagdish23** [comment]
how to use it?
